components/dashboard/src/components/forms/CheckboxInputField.tsx

```diff
@@ -56,7 +56,7 @@ export const CheckboxInput: FC<CheckboxInputProps> = ({
             <input
                 type="checkbox"
                 className={classNames(
-                    "h-4 w-4 mt-0.5 rounded cursor-pointer border-2 dark:filter-invert",
+                    "h-4 w-4 mt-1 rounded cursor-pointer border-2 dark:filter-invert",
```

Comment:
**question**: Is this change needed?

Reply:
@gtsiolis As I mentioned in the review comment above, if we increase the font size for the checkbox text label from small-sized font to medium-sized font, we need to position the checkbox square down a bit to match the checkbox label text for better UX.

See below to understand this change better -
| Checkbox with `mt-0.5` for medium-sized font | Checkbox with `mt-1` for medium-sized font |
|--------|--------|
| ![Dashboard — Gitpod - Google Chrome 31-03-2023 12_37_12 (2)](https://user-images.githubusercontent.com/62554685/229048612-53b703ce-48d3-47f2-be15-cddf278505c6.png) | ![Dashboard — Gitpod - Google Chrome 31-03-2023 12_39_29 (2)](https://user-images.githubusercontent.com/62554685/229049063-3abb497d-1e2a-46fd-858d-5c143e9df354.png) |

Reply:
Thanks for clarifying, @Devansu-Yadav! Yeah, makes sense. LGTM.